frontend/src/app/settings/modelsources/git-settings/delete-git-settings-dialog/delete-git-settings-dialog.component.html

```diff
@@ -3,15 +3,15 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<h1 mat-dialog-title>Delete Instance {{ instance.name }}</h1>
-<div mat-dialog-content>
+<h1>Delete Instance {{ instance.name }}</h1>
```

Comment:
This dialog misses some padding, so please put a div with the `dialog` class around it.